apps/nextjs-website/src/pages/[productSlug]/apis.tsx

```diff
@@ -0,0 +1,45 @@
+import { GetStaticPaths, GetStaticProps, GetStaticPropsResult } from 'next';
+import { Product } from '@/lib/types/product';
+
+import { getApi, getApisPaths } from '@/lib/api';
+
+import Hero from '@pagopa/pagopa-editorial-components/dist/components/Hero';
+import { ApiViewer } from '@/components/ApiViewer/ApiViewer';
+
+type Params = {
+  productSlug: string;
+};
+
+export const getStaticPaths: GetStaticPaths<Params> = () => ({
+  paths: getApisPaths() as string[],
+  fallback: false,
+});
+
+export type ApisPageProps = {
+  readonly product: Product;
+  readonly hero?: {
+    readonly title: string;
+    readonly subtitle: string;
+  };
+  readonly yaml: string;
```

Comment:
I expect `yaml` attribute to be named `yamlUrl` or something similar to make explicit that is not a file but an URL